pom.xml

```diff
@@ -57,6 +57,10 @@
   </scm>
 
   <repositories>
+    <repository>
+      <id>apache</id>
+      <url>https://repository.apache.org/content/repositories/public/</url>
```

Comment:
Why this change was not needed in previous releases?

Reply:
@CuriousVini Details about this change is in the JIRA - https://issues.cask.co/browse/CDAP-13570?focusedCommentId=32916&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-32916